### PR TITLE
Update/add less greater buttons to tables

### PR DESCRIPTION
**Pull request by yannick-polius**
PR to add <= and >= buttons to the analysis table in the new kibana. all functionality is as it should be, see screenshot

screenshot_20160316_150228

### CODE REVIEW

File: src/kibana/components/filter_manager/filter_manager.js [hunk 1]
```diff
@@ -65,8 +67,21 @@ define(function (require) {
...
           } else {
-            filter = { meta: { negate: negate, index: index }, query: { match: {} } };
-            filter.query.match[fieldName] = { query: value, type: 'phrase' };
+              if (range_match) {
+                  filter = { meta: { negate: negate, index: index }, range: {} };
+                  if (operation === "<=") {
+                      filter.range[fieldName] = {
+                          lte : value
+                      };
+                  } else if (operation == ">=") {
```
Comment (alexweltman) [inline src/kibana/components/filter_manager/filter_manager.js]: could also be triple equals

**Comment by yannick-polius** [comment]
@alexweltman let me know if there's a better place to put the fa css, i chose the main file because it was the only one that i could find that both already contained fa css and was related to the component i wanted to skin

File: src/kibana/styles/main.less [hunk 2]
```diff
@@ -203,7 +203,16 @@ notifications {
     z-index: 2 !important;
   }
 }
-
+.fa-search-less-than-equal:before {
+  content: "\2264";
```
Comment (alexweltman) [inline src/kibana/styles/main.less]: awesome approach! I would name them something other than 'fa' cause they aren't part of font awesome. like 
'lr-custom--search-less-than-equal' or something

File: src/kibana/styles/main.less [hunk 3]
```diff
@@ -203,7 +203,16 @@ notifications {
...
+  font-size: 16;
+  font-weight: bold;
+}
+.fa-search-greater-than-equal:before {
```
Comment (alexweltman) [inline src/kibana/styles/main.less]: same thing about having fa in the name

**Comment by alexweltman** [comment]
:+1: looks great. a few minor comments and I wanna see a demo.

File: src/kibana/components/doc_viewer/doc_viewer.html [hunk 4]
```diff
@@ -23,6 +23,16 @@
                 tooltip="Filter out value"
                 tooltip-append-to-body="1"
                 class="fa fa-search-minus"></i>
+              <i ng-click="filter(mapping[field], flattened[field],'<=')"
+                 tooltip="Filter less than or equal to value"
+                 tooltip-append-to-body="1"
+                 class="fa fa-search-less-than-equal"
```
Comment (alexweltman) [inline src/kibana/components/doc_viewer/doc_viewer.html]: what properties do you get from using the class 'fa' along with your custom class?

Comment (yannick-polius) [inline src/kibana/components/doc_viewer/doc_viewer.html]: good question, nothing useful. it actually looks a little better without it

**Comment by yannick-polius** [comment]
successfully demoed to alex, anyone else interested can look at the kibana4 repo on my box

File: src/kibana/styles/main.less [hunk 5]
```diff
@@ -203,7 +203,16 @@ notifications {
     z-index: 2 !important;
   }
 }
-
+.lr-custom-search-less-than-equal:before {
+  content: "\2264";
```
Comment (KjellKod) [inline src/kibana/styles/main.less]: I don't understand what the ```content: "\2264";``` stand for, can you explain?

File: src/kibana/styles/main.less [hunk 6]
```diff
@@ -203,7 +203,16 @@ notifications {
...
+  font-size: 20;
+  font-weight: bold;
+}
+.lr-custom-search-greater-than-equal:before {
+  content: "\2265";
```
Comment (KjellKod) [inline src/kibana/styles/main.less]: dito

**Comment by alexweltman** [comment]
@yannick-polius I think it's ok to have it in that css file since those icons could be used throughout the app, if we wanted to. However I think naming them with 'fa' is a little misleading, since it's really just a unicode symbol

File: src/kibana/styles/main.less (hunk 5 above)
Comment (KjellKod) [inline src/kibana/styles/main.less]: (Yannick explained that they were unicode characters). 

@yannick-polius please put a comment in that explains this

**Comment by yannick-polius** [comment]
@KjellKod @alexweltman updated with comments, looks good to go

**Comment by KjellKod** [comment]
:+1:

**Pull request merged**